src/applicationcomposer/messageHandlers/generateResourceHandler.ts

```diff
@@ -133,21 +132,17 @@ async function generateResource(prompt: string) {
 
         const elapsedTime = globals.clock.Date.now() - startTime
 
-        // TODO-STARLING: Reduce/remove below
-        getLogger().debug(`===== CW Chat prompt start ====`)
-        getLogger().debug(prompt)
-        getLogger().debug(`===== CW Chat prompt end ======`)
-        getLogger().debug(`===== CW Chat metadata start ======`)
-        getLogger().debug(`CW Chat conversationId = ${conversationId}`)
-        getLogger().debug(`CW Chat metadata = \n${JSON.stringify(metadata, undefined, 2)}`)
-        getLogger().debug(`CW Chat supplementaryWebLinks = \n${JSON.stringify(supplementaryWebLinks, undefined, 2)}`)
-        getLogger().debug(`CW Chat references = \n${JSON.stringify(references, undefined, 2)}`)
-        getLogger().debug(`===== CW Chat metadata end ======`)
-        getLogger().debug(`===== CW Chat raw response start ======`)
-        getLogger().debug(`${response}`)
-        getLogger().debug(`===== CW Chat raw response end ======`)
-        getLogger().debug(`CW Chat initial response time = ${initialResponseTime} ms`)
-        getLogger().debug(`CW Chat elapsed time = ${elapsedTime} ms`)
+        getLogger().debug(
+            `CW Chat Debug message:
+             prompt = "${prompt}",
+             conversationId = ${conversationId},
+             metadata = \n${JSON.stringify(metadata, undefined, 2)},
+             supplementaryWebLinks = \n${JSON.stringify(supplementaryWebLinks, undefined, 2)},
```

Comment:
instead of `JSON.stringify` you could use `%O`, then the logger will format it in a much more readable way.

    getLogger().debug('... %O ...', supplementaryWebLinks, ...)